### DISCUSSION TITLE
Any advice would be appreciated 👍

### DISCUSSION THREAD

**Discussion by Cappy-OO7** [community/community#132576] [Actions]
### Select Topic Area

Question

### Body

  If I accidentally set a repo to Public, how can I change it to Private without having anything being affected by it? I am assuming it was not meant to be public since it help private information in it. If it was, indeed, intended to be Public, then I will repost it as Public. I just need to know.
 Also...how do I run an asset script to my trust wallet without forking(merging) it to the (MAIN)public one?
 🙏 Thx

**Comment by danialtavakoli** [comment]
If you've accidentally set your repository to public and it contains private information, you should change it back to private as soon as possible to minimize any potential exposure. Here's how you can change a GitHub repository from public to private:

### Changing a GitHub Repository from Public to Private

1. **Go to your repository** on GitHub.
2. **Click on the "Settings"** tab in the repository menu.
3. Scroll down to the **"Danger Zone"** section at the bottom of the settings page.
4. Look for the option **"Change repository visibility"** and click the **"Change visibility"** button.
5. Select **"Make private"** in the dialog that appears and confirm the change.

This will make your repository private again, and only those with access permissions will be able to see it.
